expression/aggregation/agg_to_pb.go

```diff
@@ -28,11 +28,9 @@ import (
 	"github.com/pingcap/tipb/go-tipb"
 )
 
-// AggFuncToPBExpr converts aggregate function to pb.
-func AggFuncToPBExpr(sctx sessionctx.Context, client kv.Client, aggFunc *AggFuncDesc) *tipb.Expr {
-	pc := expression.NewPBConverter(client, sctx.GetSessionVars().StmtCtx)
-	var tp tipb.ExprType
-	switch aggFunc.Name {
+// GetTiPBExpr return the TiPB ExprType of desc.
+func (desc *baseFuncDesc) GetTiPBExpr(tryWindowDesc bool) (tp tipb.ExprType) {
+	switch desc.Name {
```

Comment:
GetTiPBExpr -> GetTiPBExprOfAgg  ??

Reply:
it not only can get agg desc, but also can get window desc